app/styled-components/Spinner.tsx

```diff
@@ -0,0 +1,26 @@
+import styled, { keyframes } from 'styled-components';
```

Comment:
there's a "components" folder in /app.   let's stick this in there unless you have objections

Reply:
I moved this. Fixed.